map/map-reference-scale/README.md

```diff
@@ -1,35 +1,35 @@
 # Map reference scale
 
-Set the map's reference scale and choose which feature layers should honor the reference scale.
+Set the map's reference scale and which feature layers should honor the reference scale.
 
-![](MapReferenceScale.png)
+![Image of map reference scale](MapReferenceScale.png)
 
 ## Use case
 
 Setting a reference scale on an `ArcGISMap` fixes the size of symbols and text to the desired height and width at that scale. As you zoom in and out, symbols and text will increase or decrease in size accordingly. When no reference scale is set, symbol and text sizes remain the same size relative to the `MapView`.
 
-Map annotations are typically only relevant at certain scales. For instance, annotations to a map showing a construction site are only relevant at that construction site's scale. So, when the map is zoomed out that information shouldn't scale with the `MapView`, but should instead remain scaled with the `ArcGISMap`.
+Map annotations are typically only relevant at certain scales. For instance, annotations to a map showing a construction site are only relevant at that construction site's scale. So, when the map is zoomed out that information shouldn't scale with the `MapView`, but should instead remain scaled with the `ArcGISMap`. 
 
 ## How to use the sample
 
-* Use the drop down menu at the top left to set the map's reference scale.
-* Click the "Set Map Scale to Reference Scale" button to set the map view scale to that of the map reference scale.
-* Use the top right menu checkboxes to apply the map reference scale to the map's feature layers (which should scale according to the reference scale).
+* Use the drop box at the top to set the map's reference scale (1:500,000 1:250,000 1:100,000 1:50,000).
+* Click the button to set the map scale to the reference scale.
+* Use the menu checkboxes in the layer menu to set which feature layers should honor the reference scale.
 
 ## How it works
 
-* Set the map reference scale property on the `ArcGISMap` with `map.setReferenceScale(double)`.
-* Set the scale symbols property on each individual `FeatureLayer` within the map with `featureLayer.setScaleSymbols(boolean)`.
+1. Get and set the reference scale property on the `ArcGISMap` object.
```

Comment:
This "object" stuff is just bad IMO

Reply:
this would have been an early revamped readme when we decided to use objects and then the team also decided it was "just bad" and stopped using it :p